migrations/011-update-article-rating-column.js

```diff
@@ -0,0 +1,26 @@
+const tableName = 'Articles';
```

Comment:
Should this migration not just be about adding the `aveRating` column? Is there a need for doing stuff with the rating column that has already been done in the earlier migration?

Reply:
I also changed the `rating` dataType to JSON, and sequelize returns an error of `cannot automatically update dataType to JSON` when I try to update just the dataType without deleting the column and re-adding the column.

Reply:
Can you check the migration file for a solution to the automatic casting?

@habibaudu Did something like that in the `009-create-articles-update-tag.js` migration file

Reply:
I've tried this, and I still get the error `cannot cast type integer to json`

Reply:
Okay. In that case, go ahead with this. I made the comment earlier majorly because i saw two different migration files. I see one now, and that's fine by me.

Reply:
Okay, thanks